sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/ReviewSqlQueriesForSecurityVulnerabilitiesTest.cs

```diff
@@ -44,5 +53,90 @@ public void ReviewSqlQueriesForSecurityVulnerabilities()
                 null,
                 Verifier.SystemDataAssembly, OracleManagedDataAccessClientAssembly, SystemDataSqlServerCeAssembly);
         }
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void ReviewSqlQueriesForSecurityVulnerabilities_GenerateProtobuf()
+        {
+            var workDir = Path.Combine(TestContext.TestRunResultsDirectory, "0");
+            Directory.CreateDirectory(workDir);
+
+            Verifier.VerifyCSharpAnalyzer(@"
+class Program
+{
+    string Property2
+    {
+        set { var x = value; }
```

Comment:
It is beyond me why the setter is not walked when it is defined after the getter.

Reply:
@valhristov what does this comment mean ? is there an issue in Roslyn visit ? or in our analyzer code ? if the latter : do we have a ticket for it ?